pkg/useragent/analyzer.go

```diff
@@ -60,21 +60,23 @@ func (a *analyzer) start() {
 			// set up writer output
 			var output update
 
-			if len(datum.OrigIps) > 10 {
-				datum.OrigIps = datum.OrigIps[:10]
+			origIPs := datum.OrigIps.Items()
```

Comment:
Note that .Items() must be called to convert the map into an array. 

https://github.com/activecm/rita/commit/cd8dd38af38d30b06b0141d08b3370435ef2ba4e
https://github.com/activecm/rita/commit/180de847eb6ddb15e008891d9d690427ef307b45